dashboard/src/main/java/org/apache/uniffle/dashboard/web/JettyServerFront.java

```diff
@@ -83,14 +88,31 @@ private void initialization() {
   }
 
   private void setRootServletHandler() {
-    final ContextHandler contextHandler = new ContextHandler("/");
+    HandlerList handlers = new HandlerList();
+    ResourceHandler resourceHandler = addResourceHandler();
+    String coordinatorWebAddress = conf.getString(DashboardConf.COORDINATOR_WEB_ADDRESS);
+    boolean sslEnable = conf.getBoolean(DashboardConf.JETTY_SSL_ENABLE, false);
```

Comment:
Reusing `JETTY_SSL_ENABLE` is weird. Could you add a new parameter? Such as `coodinator.ssl.enable`. Or change the format of `COORDINATOR_WEB_ADDRESS`  to `http://host:port/`